platform/backend/core/src/main/kotlin/io/hamal/core/adapter/Func.kt

```diff
@@ -63,17 +63,17 @@ interface FuncUpdatePort {
 
 }
 
-interface FuncPort : FuncCreatePort, FuncDeployPort, FuncGetPort, FuncInvokePort, FuncListPort, FuncUpdatePort,
+interface FuncCodePort : FuncCreatePort, FuncDeployPort, FuncGetPort, FuncInvokePort, FuncListPort, FuncUpdatePort,
```

Comment:
Why did you rename the interface and the implementation? I think it should still be Func*

Reply:
Forgot to delete, that was a bad idea